README.md

```diff
@@ -26,7 +26,7 @@ On the other hand, Google maintains a branch that uses cgo and BoringSSL to impl
 
 ### Multiple OpenSSL versions supported
 
-The `openssl` package has support for multiple OpenSSL versions, namely 1.0.1, 1.1.0, 1.1.1 and 3.0.2.
+The `openssl` package has support for multiple OpenSSL versions, namely 1.0.2, 1.1.0, 1.1.1 and 3.0.2.
```

Comment:
Hmm... should 3.0.2 be changed to 3.0.1 also, to match what CI tests?

https://github.com/microsoft/go-crypto-openssl/blob/2cec4414578a6f4c7bdcdbb424c15774615bb45f/.github/workflows/test.yml#L9-L10

(This might also explain the typo--patch versions swapped between these two.)

Reply:
We should support all OpenSSL 3.0 versions at once, in theory the new versioning policy forbides API breakages between patch versions. Having said this, it might be good to test with OpenSSL 3.0.0 and OpenSSL 3.0.8 (the latest at this moment).